### DISCUSSION TITLE
how to call custome js function after katex complete it's job?

### DISCUSSION THREAD

**Discussion by Rss-Imran** [KaTeX/KaTeX#3008] [Q&A]
how to call custome js function after katex complete it's job?

**Comment by edemaine** [comment]
You can just call the function after calling `renderMathInElement` (assuming you're calling that), as it's synchronous.

**Reply by Rss-Imran** [reply]
suppose 
p id="body" style="visibility:hidden;">
       $$ \sum_{i=0}^n i^2 = \frac{(n^2+n)(2n+1)}{6} $$
p

after finish katex rendering how to call
document.getElementById("body").style.visibility="visible";
could i have an example about it?

**Reply by edemaine** [reply]
Something like this?
```js
renderMathInElement(document.getElementById("body"));
document.getElementById("body").style.visibility="visible";
```

**Reply by Rss-Imran** [reply]
<script defer src="katex/contrib/auto-render.js"  onload="renderMathInElement(document.body); document.getElementById('body').style.visibility='visible';"></script> 
works for me.
